templates/pulp-apache.conf.erb

```diff
@@ -10,7 +10,7 @@ Alias /pub /var/www/html/pub
   Require all granted
 </Location>
 
-Include /etc/pulp/vhosts80/*.conf
+Include /etc/httpd/conf.d/vhosts80/*.conf
```

Comment:
`puppet-pulp` uses `${apache::confd_dir}`, but here we hardcode `/etc/httpd/conf.d`? (Debian ships httpd config in `/etc/apache2` ;))

Reply:
this comment remains